### ISSUE TITLE
Sourcemaps do not work since 0.13

### ISSUE THREAD

**Issue by CxRes** [nwjs/nw.js#4730]
Post v0.13 (tried 0.13.2 - 0.14.0, windows 10), When using a combined file and source maps I get red icon folder and empty files when I look at sources in the dev tools. 

This is not a problem with 0.12.3, source files load perfectly there for the same code.

**Comment by rogerwang** [comment]
Could you please provide a sample? I tried https://github.com/NETTUTS/Source-Maps-101 and it works for me.

**Comment by CxRes** [comment]
Give me a little time, this is occurring in ~50KLOC file, I'll try to send you a smaller example when I get a little free.

**Comment by rogerwang** [comment]
Please reopen this when there is more information.

**Comment by CxRes** [comment]
Try running the project in here. Sourcemap works in 0.12.3 but not later...
[des.zip](https://github.com/nwjs/nw.js/files/233439/des.zip)

**Comment by CxRes** [comment]
Please Reopen....

**Comment by CxRes** [comment]
Just checking if there has been any progress on this?

**Comment by CxRes** [comment]
Is there any intention by the developers to fix this? @rogerwang

**Comment by rogerwang** [comment]
Will check this soon

**Comment by CxRes** [comment]
@rogerwang Thanks! I was getting worried... sorry to keep pestering you :) !!!

**Comment by CxRes** [comment]
Sorry, but it is August already, what is the ETA on this bugfix????

**Comment by CxRes** [comment]
Sorry, but it is August already, what is the ETA on this bugfix???? @rogerwang

**Comment by rogerwang** [comment]
Your example is working for me with 0.16.0.

**Comment by CxRes** [comment]
I just downloaded the example I submitted to you to check if I made some mistake. It turns out sourcemaps are not only not working in 0.16.0 were not working in 0.12.3 either (0.12.3 was picking up `src` folder which was alongside the `des` folder when I had tested previously. This time there was no `src` folder since I used the downloaded package in its own folder and I get file not found error in 0.12.3). 

Please try with Windows x64... it does not work for me at all. Look at `src/app/app.js` or `src/bootstrap.js` in Dev Tools > Sources. Both these files come up as empty and the `src` and `src/app` are in red color instead of blue. Sourcemaps are detected because the folder structure is recreated correctly.

**Comment by CxRes** [comment]
After a lot more testing with Chrome, it seems this is definitely a nwjs (probably Windows only) problem. 

1. Sourcemaps are loading perfectly in Chrome in all tests (Chrome 52.0.2743.82).
2 nwjs works perfectly when sourcemaps is a self contained bundle ie includes the source file within itself. However, it fails as described above, when the sourcemap does not include the code from the minified files.

**Comment by Christywl** [comment]
I can reproduce this issue with _test.zip_ file on Linux/Windows with nwjs-sdk-v0.21.6. `src/app/app.js` and `src/bootstrap.js` are empty in `Dev Tools > Sources`.

**Comment by rogerwang** [comment]
@Christywl the .js files are missing in the attachment. And I didn't see it in Chrome browser as well if I open index.html from Chrome.

**Comment by Christywl** [comment]
Yes, I just check, there are no src/*.js files in the zip file. 
I saw https://github.com/nwjs/nw.js/issues/4730#issuecomment-237487969 before. 

> Please try with Windows x64... it does not work for me at all. Look at src/app/app.js or src/bootstrap.js in Dev Tools > Sources. Both these files come up as empty and the src and src/app are in red color instead of blue. Sourcemaps are detected because the folder structure is recreated correctly.

So need the reporter provide the full files to double check?

**Comment by CxRes** [comment]
@Christywl I had provided the exact files which caused a problem - nothing was done!!!
